cardano-node-chairman/app/Main.hs

```diff
@@ -0,0 +1,161 @@
+{-# LANGUAGE NamedFieldPuns #-}
```

Comment:
Should rename that to maybe `chariman.hs`. The leading lower case shows that it its a Haskell main and not a module.

Reply:
Should I use `cardano-node-chairman.hs` for now, since that's the name of the exe?